ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -497,6 +497,8 @@ def odf_storagesystems_ui(self):
         if not (
             config.DEPLOYMENT.get("external_mode")
             or config.ENV_DATA["mcg_only_deployment"]
+            or config.ENV_DATA["platform"].lower()
+            == constants.HCI_PROVIDER_CLIENT_PLATFORMS
```

Comment:
There should be `in` instead of `==`, right?

Reply:
yes, you are right. Thanks

Reply:
fixed